packages/dom/src/tabbable.js

```diff
@@ -96,7 +102,7 @@ function mapElementToObjectTabbable( element, index ) {
  * An array map callback, returning an element of the given mapped object's
  * element value.
  *
- * @param {Object} object Mapped object with index.
+ * @param {{ element: Element }} object Mapped object with index.
```

Comment:
I know this was like that before, but did we mean this instead:

```suggestion
 * @param {{ element: Element }} object Mapped object with element.
```

Reply:
Yes, we did mean that I believe 🙂